apps/assisted_tagging/filters.py

```diff
@@ -1 +1,31 @@
-#
+import django_filters
+
+from deep.filter_set import generate_type_for_filter_set
+from .models import DraftEntry
+from utils.graphene.filters import IDFilter, MultipleInputFilter
+from .enums import (
+    DraftEntryTypeEnum
+)
+
+
+class DraftEntryFilterSet(django_filters.FilterSet):
+    leads = IDFilter(field_name='lead')
+    draft_entry_types = MultipleInputFilter(DraftEntryTypeEnum, field_name='draft_entry_type')
+    is_discarded = django_filters.BooleanFilter(method='filter_discarded')
+
+    def filter_discarded(self, queryset, name, value):
+        if value:
+            return queryset.filter(is_discarded=value)
+        return queryset.filter(is_discarded=False)
```

Comment:
Let's do this.. If value is somehow None.
```suggestion
    def filter_discarded(self, queryset, name, value):
        if value is None:
            return queryset
        return queryset.filter(is_discarded=value)
```

Also, why are we defining this instead of just using field_name?